pywbem/cim_operations.py

```diff
@@ -2094,15 +2164,15 @@ def _get_rslt_params(self, result, namespace):
                 rtn_objects = p[2]
 
         if not end_of_sequence_found and not enumeration_context_found:
-            raise CIMError(
-                CIM_ERR_INVALID_PARAMETER,
-                "EndOfSequence or EnumerationContext required in response.",
+            raise ParseError(
+                "Expecting EndOfSequence or EnumerationContext output "
+                "parameter in open/pull response, did not get any of them",
```

Comment:
how about change did not get any of them to "but none received"

Reply:
Also, maybe Expected rather than Expecting.

Reply:
DONE.